apps/web/pages/booking/[uid]/join/[hash].tsx

```diff
@@ -0,0 +1,88 @@
+import type { GetServerSideProps } from "next";
+import { z } from "zod";
+
+import { BbbApi, hashAttendee } from "@calcom/app-store/bigbluebutton/lib";
+import { bbbOptionsSchema } from "@calcom/app-store/bigbluebutton/lib/bbbApi";
```

Comment:
Thanks for the PR @abnud11 
 Concerns were raised in the previous pr about importing apps in core modules here https://github.com/calcom/cal.com/pull/10803#pullrequestreview-1652440876.
Have you tried if there is any other way around this?